src/components/BulkEditPane/BulkEditListResult/BulkEditInApp/BulkEditInApp.js

```diff
@@ -8,6 +8,7 @@ import {
   Layout,
 } from '@folio/stripes/components';
 
+import { useState } from 'react';
```

Comment:
Please move third party imports to the top of the file

Reply:
Fixed